app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class TasksController < ApplicationController
+  layout false
```

Comment:
Why no layout?

Reply:
The layout will not be needed in this case.